src/Post/Post.css

```
@@ -359,3 +366,62 @@ section.comment-input-section {
   text-decoration: underline;
   cursor: pointer;
 }
+.img-buttons-container {
+  position: relative;
+  top: -50%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 0;
+  background: rgba(0, 0, 0, 0.1);
+}
+.next-img-button,
+.prev-img-button {
+  padding: 0;
+  margin: 8px;
+  border-radius: 50%;
+  height: 23px;
+  width: 23px;
+  border: none;
+  background: transparent;
+  box-shadow: 2px 2px 6px 1px rgba(0, 0, 0, 0.28);
+}
+.next-img-button.hidden,
+.prev-img-button.hidden {
+  visibility: hidden;
+}
+.next-img-button svg,
+.prev-img-button svg {
+  opacity: 0.7;
+}
+.progress-dots-section {
+  display: flex;
+  gap: 5px;
+  justify-content: center;
+  margin-top: 15px;
+}
+.progress-dots-section.is-expanded {
+  display: flex;
+  gap: 5px;
+  justify-content: center;
+  margin: 0;
+  margin-top: -45px;
+  z-index: 2;
+  position: relative;
+}
```

Since `is-expanded` addes more CSS, you don't need to duplicate the code from `.progress-dots-section`
```suggestion
.progress-dots-section.is-expanded {
  margin: 0;
  margin-top: -45px;
  z-index: 2;
  position: relative;
}
```

---

Done